hack/generate-unified-csv.sh

```diff
@@ -60,6 +60,7 @@ $CSV_MERGER \
 	--noobaa-core-image="$NOOBAA_CORE_IMAGE" \
 	--noobaa-db-image="$NOOBAA_DB_IMAGE" \
 	--ocs-image="$OCS_IMAGE" \
+	--ocs-must-gather-image="$OCS_MUST_GATHER_IMAGE" \
```

Comment:
Not sure if `OCS_MUST_GATHER` should be added to the check for required env vars (and a few more, fwiw...)

Reply:
must-gather is a optional flag. so i guess no need.

Reply:
I don't fully understand this. How is must-gather optional? The script unconditionally specifies `--ocs-must-gather=....` and if the VAR is not set, I gues this command would fail? That is required, imho, and it applies to all parameters, but I must be missing something.

Reply:
But I agree that this is a seprate topic. not part of this PR. :-)